docs/adding games.md

```diff
@@ -36,8 +38,9 @@ strictly required
 * Send a status update packet alerting the server that the player has completed their goal
 
 Libraries for most modern languages and the spec for various packets can be found in the
-[network protocol](/docs/network%20protocol.md) API reference document.
+[network protocol](network%20protocol.md) API reference document.
 
+(world)=
```

Comment:
This (and other references like this) will render on GitHub and probably other markdown viewers, so I don't think this is great. Is there an alternative or do we need string replace in the sphinx script to have two variations of the file?

Reply:
as far as i could tell this is the only way to do doc linking with myst. string replace is probably the better approach but unsure when i'll get to it